docs/user/set-up/quick-starts.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases:
+- /docs/agent/latest/set-up/quick-starts/
+title: Grafana Agent quick starts
```

Comment:
Are we using the term 'quick starts' anywhere else in our docs?

Reply:
i copied it from cloud...but will chat to team and get back

Reply:
any other suggestions welcome :)